db/migrate/20160331232622_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```

Comment:
Then I think we want to continue to have this file in (we should re-add it to the branch).  To get the migrations to work on your system it probably makes sense to rebuild the database with the migrations only on the branch.  Here's how you can do that.

First, stop your server and any rails consoles. 

Second, run this command (and make sure you don't see any error messages when you run it):

```
rake db:drop
```
 
Third, run this command

```
rake db:create
```

Fourth, run this command:

```
rake db:migrate
```